tests/unit/store/waveform.spec.js

```diff
@@ -176,13 +179,13 @@ describe("store/waveform", () => {
         context
       );
       await bound_ping_get_waveform_data();
-      expect(mocked_axios.history.get.length).toBe(1);
+      expect(mocked_axios.history.get).toHaveLength(1);
       expect(mocked_axios.history.get[0].url).toMatch(
         "http://localhost:4567/get_available_data?currently_displayed_time_index=" +
           expected_idx
       );
     });
-    test("verify if the get_available_data is invoked with http error response empty data its handled ", async () => {
+    test("When the get_available_data is invoked, Then http error response of empty data its handled", async () => {
       mocked_axios
         .onGet(get_available_data_regex) // We pass in_simulation_mode true and validate default false is replaced
```

Comment:
these comments appear to be copy/pasted from somewhere else and don't seem to be relevant here.  is that correct?

Reply:
it does not look like the code was updated